ci-operator/config/redhat-developer/rhdh/.claude/skills/rhdh-ocp-lifecycle/scripts/check-ocp-lifecycle.sh

```
@@ -0,0 +1,217 @@
+    exit 1
+  fi
+
+  echo "=== OCP Lifecycle ==="
+  echo ""
+  printf "%-10s %-10s %-10s %-35s %-12s %-12s\n" \
+    "VERSION" "OCP_SUPP" "RHDH_SUPP" "PHASE" "GA_DATE" "END_DATE"
+  printf "%-10s %-10s %-10s %-35s %-12s %-12s\n" \
+    "-------" "--------" "---------" "-----" "-------" "--------"
+
+  # Build lookup for RHDH-supported OCP versions
+  RHDH_OCP_SET=$(echo "$RHDH_SUPPORTED_OCP" | tr '\n' '|')
```

Shellcheck (0.11.0)

[warning] RHDH_OCP_SET appears unused. Verify use (or export if used externally).

(SC2034)